cubic-server/types.hpp

```diff
@@ -212,4 +212,18 @@ struct std::hash<Position2D> {
     }
 };
 
+struct Tps {
+    float oneMinTps;
+    float fiveMinTps;
+    float fifteenMinTps;
+    std::string toString() const;
+};
+
+struct MSPTInfos {
+    float mean;
+    float min;
```

Comment:
You need to put the min first as in-game the min is higher than the average :)

Reply:
See the screenshot:
![image](https://github.com/CubicMC/cubic-server/assets/44233177/2804bb36-c320-4c97-9cb4-58311ebd5fe0)